File: Easy/Java/27. Remove Element.java
class Solution {
    public int removeElement(int[] nums, int val) {
        // Initialize a count to keep track of elements
        int count = 0;
        
        // Go over the array, shifting leftward if a value
        // is found and is the same as val
        for(int i = 0; i < nums.length; i++){
            if(nums[i] != val){
                nums[count] = nums[i];
                count++;
            }
        }
        
        // Return the count of elements left in the array
        return count;
    }
}

File: Easy/Python/27. Remove Element.py
class Solution(object):
    def removeElement(self, nums, val):
        # Intialize a count to keep track of elements
        count = 0
        
        # Go over the array, shifting leftward if a value
        # is found and is the same as val
        for i in range(len(nums)):
            if(nums[i] != val):
                nums[count] = nums[i]
                count += 1
        
        # Return the count of elements left in the array
        return count
